### PR TITLE
Define align-center class for img blocks (#311)

### LINKED ISSUE #311: the `align: center` setting is not respected for images
We set the alignment for images using the `align: center` setting but this setting is not applied. For instance, the following block...
```rst
.. image:: /images/installation/install/update-arrow.png
   :align: center
```
... results in:
image

Upon inspection of the related HTML block, we can see that the style was correctly applied:
```html
<img alt="../_images/update-arrow.png" class="align-center" src="../_images/update-arrow.png">
```

What is missing is actually the definition of the `align-center` class in our css stylesheet.

It is thus proposed to add such definition in our stylesheet by copying its current definition in the sphinx *style* that we currently inherit from:
```css
img.align-center, figure.align-center, .figure.align-center, object.align-center {
  display: block;
  margin-left: auto;
  margin-right: auto;
}
```
https://github.com/sphinx-doc/sphinx/blob/9e1b4a8f1678e26670d34765e74edf3a3be3c62c/sphinx/themes/basic/static/basic.css_t#L292-L296

### PR DESCRIPTION
**Pull request by FroggyFlox**
Fixes #311 
@phillxnet, ready for review.

### This pull request's proposal
As detailed in #311, we currently are lacking a definition for the `align: center` option in our `.. image::` directive blocks. This pull request (PR) simply adds that definition.
This definition was taken from Sphinx's basic theme stylesheet:
https://github.com/sphinx-doc/sphinx/blob/9e1b4a8f1678e26670d34765e74edf3a3be3c62c/sphinx/themes/basic/static/basic.css_t#L292-L296

### Checklist
- [x] With the proposed changes no Sphinx errors or warnings are generated.
- [x] I have added my name to the AUTHORS file, if required (descending alphabetical order).

### CODE REVIEW

**Review by phillxnet — COMMENTED** [review]
@FroggyFlox Nice find.
But I have a potential anomaly with testing this in that if I remove the style="width: 100%;" from the example image given in the issue (via direct edit within dev tools) it ends up displaying in the middle anyway (chrome & firefox) and I see the following css referenced in the element breakdown from docs/_static/basic.css:
```
img.align-center, .figure.align-center, object.align-center {
  display: block;
  margin-left: auto;
  margin-right: auto;
}
```
This is via our currently published docs. I'm likely missing something on this one.

I do see that post pr the same css property "img.align-center" is then picked up from:
style.css as per your addition.

The heading in the basic.css is as follows:
```
/*
 * basic.css
 * ~~~~~~~~~
 *
 * Sphinx stylesheet -- basic theme.
 *
 * :copyright: Copyright 2007-2020 by the Sphinx team, see AUTHORS.
 * :license: BSD, see LICENSE for details.
 *
 */
```
All tests via Apache rendering.

I'm a little reluctant to add a style element that we have already, especially given the predominant write only nature of css, i.e. we tend to only add / override rather than correct given the difficulty of tracking stuff down.

What am I missing here?

I did however manage to reproduce your results if I rendered via the python server
```
python3 -m http.server 8000
```
Not the 3 in the above was required as my default Python is still 2.

## Before pr
before-pr312

## After pr
after-pr312

So in short, our current master, when rendered via Apache, finds an existing img.align-center 
from basic.css from  "* Sphinx stylesheet -- basic theme." where as this is not found when using the Python3 webserver.

Again this could still be down to a bug in our buildbot, possibly leaving older stuff in place or the like. But prior to merging this I'd like to be sure given the house of cards css ends up being. And if we add this admittedly trivial addition it may lead to further potentially redundant additions and we add further to the stack of redundant theme elements.

Apologies if I have this all wrong and am not seeing the obvious here.

**Comment by FroggyFlox** [comment]
Thanks for catching that, @phillxnet , it must be resulting from apache vs http.server. 
I'll have a closer look later and confirm what you see... I couldn't see the sourcing of basic.css on my end for some reason and it might be due to http.server.

Sorry for the borked PR here :-\

**Comment by FroggyFlox** [comment]
Hi @phillxnet 
Thanks again for proper review and finding that one out... I somehow searched for this basic.css but couldn't see it, thus thinking we needed to import it, etc....
You are absolutely correct and we don't need this PR. I'm still unsure why `http.server` doesn't but that's something for another day, I'm afraid.
